### PR TITLE
Polish translation

### PR DESCRIPTION
**Pull request by nolt**
This PR/Translation is about #90.

Complete UI translation, there are few help lines left to translate in settings.cson.

**files translated**:
  - [x] `def/pl/about.cson`
  - [x] `def/pl/context.cson`
  - [x] `def/pl/menu_darwin.cson`
  - [x] `def/pl/menu_linux.cson`
  - [x] `def/pl/menu_win32.cson`
  - [ ] `def/pl/settings.cson` - translation in progress
  - [x] `def/plwelcome.cson`

**validation**
  - [ ] I've validated my translation locally by running

`npm run validation -- --locale pl`

Not validated because of errors ('atom-i18n' is not in the npm registry).

### CODE REVIEW

**Comment by liuderchi** [comment]
@nolt thanks for your contribution :tada: 
Please feel free to notify me when you think it's ready to merge. :100:

**Comment by nolt** [comment]
@liuderchi translation is ready to merge now. When I finish translating the las file I will make another PR.

thanks!

File: def/pl/menu_win32.cson [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+        value: "Szukaj pliku"
+      "Find Modified File":
+        value: "Szukaj modyfikowanego pliku"
+  "&Packages":
+    value: "Pakiety(&P)"
+  "&Help":
+    value: "Pomoc(&H)"
+    submenu:
+      "View &Terms of Use":
+        value: "Pokaż warunki korzystania"
```
Comment (liuderchi) [inline def/pl/menu_win32.cson]: Please add *hotkey hint prompt* `&` here. 
You may select whichever letter you feel comfortable with.

File: def/pl/menu_win32.cson [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+      "View &License":
+        value: "Pokaż licencję"
```
Comment (liuderchi) [inline def/pl/menu_win32.cson]: Please add *hotkey hint prompt* `&` here. 
You may select whichever letter you feel comfortable with.

**Comment by nolt** [comment]
File updated, hotkeys added.

**Comment by liuderchi** [comment]
LGTM and let's merge it : )

**Pull request merged**